### DISCUSSION TITLE
Parse query params in admin app collection content view

### DISCUSSION THREAD

**Discussion by viters** [directus/directus#16776] [Feature Requests]
Let's say I have collection items `directus.com/admin/content/items`. I can go to that page and filter the collection using search box.

I would like to populate the search box filter with the same query params that are used by Directus in queries later, like `directus.com/admin/content/items?filter[_and][0][_and][0][relation1][relation2][id][_nnull]=true`.

That would e.g. allow to create `button links` (fields) that lead to specific filtered list views.

Example use case:
I have event, which has multiple panels, panels have multiple time slots, and each time slot can have participants. I would like to create an easy way for content creator to jump to view, where he sees all participants for given event. Right now I can create bookmark or explain how filtering works - but still administrator needs to copy the ID by himself and it is overall inconvenient.
Happy path would be `Show participants` button link on event's details that leads to `directus.com/admin/content/participants?filter[_and][0][_and][0][timeslot][panel][event][id][_eq]=PASSED_VAL`.

I guess it does not need to be synced with search box later. Query params could be used only in initial load.

**Comment by br41nslug** [comment]
There is a PR in the works for this https://github.com/directus/directus/pull/14081

**Reply by viters** [reply]
Hmmm, it's worrying that it takes so long to merge. This would be really nice feature to prepare workflows for content editors.

**Reply by br41nslug** [reply]
It happens sometimes with PRs that weren't discussed beforehand 😬

**Reply by viters** [reply]
@br41nslug 
Can I do something to help?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
